protocols/identify/src/handler.rs

```diff
@@ -42,21 +43,27 @@ use smallvec::SmallVec;
 use std::collections::HashSet;
 use std::{io, task::Context, task::Poll, time::Duration};
 
+const STREAM_TIMEOUT: Duration = Duration::from_secs(60);
+const MAX_CONCURRENT_STREAMS_PER_CONNECTION: usize = 4 * 10;
```

Comment:
What is the rational for `40` streams? This seems rather large to me. In what scenario would one have `40` `libp2p-identify` streams? I would expect `5` to suffice. Playing safe, we can do `10`.

Reply:
Ok, I'll change it to `10`